packages/odyssey-react/src/components/Icon/_icon-template.js

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const changeCase = require('change-case');
+const generate = require('@babel/generator').default;
+
+function odysseyIconTemplate(
+  { template },
+  opts,
+  { componentName, jsx },
+) {
+  const plugins = ['jsx']
+  if (opts.typescript) {
+    plugins.push('typescript')
+  }
+  const typeScriptTpl = template.smart({ plugins, preserveComments: true })
+
+  const currentYear = new Date().getFullYear();
+  const headerComment = `/*!
+ * Copyright (c) ${currentYear}-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+`;
+
+const compName = componentName.name.substring(3);
+const title = changeCase.capitalCase(compName);
```

Comment:
can we write a simple util to handle this for us rather than installing a dep for every install and CI build? Might be as simple as this:

```
function capitalCase(compName) {
  const parts = compName.match(/[A-Z][a-z]+|\w+/g)
  const adjustedParts = parts.map(part => `${part.charAt(0).toUpperCase()}${part.substr(1)}`)
  return adjustedParts.join(' ')
}
```

Reply:
I'd trade the 9k download of a well tested library pretty much any day

Reply:
hmmm for a complicated use case I would agree 100%! However, for something straight forward like string manipulation that we can grok ourselves I'm not as confident pulling in a new dependency is the right move.

Actually, taking a step back we don't want to have this code automatically generated at all because the title text is not extracted for an i18n translation correctly! So  - we can remove this code entirely.